src/ApplicationCore/Entities/CatalogItem.cs

```diff
@@ -54,5 +55,15 @@ public void UpdateType(int catalogTypeId)
             Guard.Against.Zero(catalogTypeId, nameof(catalogTypeId));
             CatalogTypeId = catalogTypeId;
         }
+
+        public void UpdatePictureUri(string pictureName)
+        {
+            if (string.IsNullOrEmpty(pictureName))
+            {
+                PictureUri = string.Empty;
+                return;
+            }
+            PictureUri = $"images\\products\\{pictureName}?{new DateTime().Ticks}";
```

Comment:
Is this just for cache busting? Is it necessary?

Reply:
yes very necessary image will not refresh in the edit sfter change if this not there.